data/src/main/kotlin/com/acon/data/dto/request/GoogleTokenRequest.kt

```diff
@@ -0,0 +1,11 @@
+package com.acon.data.dto.request
+
+import com.acon.domain.type.SocialType
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class GoogleTokenRequest(
+    @SerialName("socialType") val socialType: SocialType,
+    @SerialName("idToken") val idToken: String,
```

Comment:
Response는 Nullable하게 만들어주세용
```suggestion
    @SerialName("socialType") val socialType: SocialType?,
    @SerialName("idToken") val idToken: String?,
```